### ISSUE TITLE
Add a folder-picker to the "failed to find SDK" messages to let the user just browse to it

### ISSUE THREAD

**Issue by DanTup** [Dart-Code/Dart-Code#628]
Add a button to each SDK failure message that says "Locate SDK" that opens a folder picker, validates the users selection looks like the correct SDK and then writes it to `dart.sdkPath` or `dart.flutterSdkPath` and reloads).

**Comment by DanTup** [comment]
There's a new logger API in Code but it's only proposed so can't currently use it.

**Comment by DanTup** [comment]
If invoked from the `Create Project` command, we should see if we can re-invoke that.

**Comment by DanTup** [comment]
> If invoked from the `Create Project` command, we should see if we can re-invoke that.

Can't make this work perfectly because we need to reload and then invoke the command (which isn't possible). Tried to just call `activate` manually, but since we already registered `flutter.createProject` to handle the error we end up with an error trying to register it a second time.

For now, we'll just prompt the user that we will reload the window and then they will need to re-invoke the command.
